### DISCUSSION TITLE
First Time Installation Issue

### DISCUSSION THREAD

**Discussion by dmarsman** [pH7Software/pH7-Social-Dating-CMS#1031] [Troubleshooting] [answered]
Hi, I've installed pH7 on my hosting server twice and the installation completes each time, but the website never loads after initial installation. Just getting the generic Error 500 - The website cannot handle the request. The same error while trying to load the /admin123 page. I didn't use softaculous, but instead FTP'd the files to my server and set all chmod permissions as indicated in the instructions. If I put the _install folder back in place (or don't delete it at all), the page loads and recognizes that it is there and wants me to delete it. As soon as I delete it, back to the Error 500. I can't find any log files that would tell me where to start. Any help or suggestions?

**Comment by dmarsman** [comment]
As an update, I tried installing via Softaculous and the install completed successfully, but the same error happens while loading the webpage.

**Reply by savvyvisions** [reply]
I tried uploading the software via Softaculous and all I got was a blank page.

**Reply by spiderwebb1965** [reply]
Exactly the same for me now trying to install manually gets to Step 2 and will not go any further.

**Comment by coolyou11** [comment, marked as answer]
I had the same problems trying to install this script with no support to help me. I would advise you to find an alternative script to build your site. If you don`t know how to code in PHP stay away from this script. The guy that built this script is not interested in giving any support, so it seems.

**Reply by dmarsman** [reply]
Thanks. I realized the same and moved on. This seems like it was a good script, but it is dead.

**Reply by roland1509** [reply]
If you are experiencing problem running this script, the most likely cause of it would be that Apache service is not ON in your services.msc, so its either you dont have it installed or its not started. Try to work on fixing this and rerun the scripts. This should help. If you are not able to do this, let me know , I will give you a very simple step by step way to do this, It pretty easy. and i am sure it would resolve your issue.

**Comment by Picofrank** [comment]
Same thing happened to me. You didn't installed the php rewrite module.

**Comment by hugoLguerra** [comment]
A question about a good and affordable decent hosting to install this system?
thanks since now
